src/test/java/com/mael/ttt/GameTest.java

```diff
@@ -1,50 +1,52 @@
 package com.mael.ttt;
 
-import com.mael.ttt.players.HumanPlayer;
-import com.mael.ttt.ui.SpyConsole;
-import com.mael.ttt.ui.UserInterface;
+import com.mael.ttt.players.FakePlayer;
+import com.mael.ttt.ui.UserInterfaceSpy;
+
 import org.junit.Before;
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 import static com.mael.ttt.Mark.*;
+import static org.junit.Assert.assertTrue;
 
 public class GameTest {
 
-    private int size;
     private Board board;
-    private SpyConsole spy;
     private Game game;
-    private UserInterface gameUI;
+    private UserInterfaceSpy uiSpy;
 
     @Before
     public void setUp() {
-        size   = 3;
-        board  = new Board(size);
-        spy    = new SpyConsole();
-        gameUI = new UserInterface(spy);
-        game   = new Game(new Turn(board, new BoardChecker(board), gameUI),
-                 new HumanPlayer(gameUI, PLAYER),
-                 new HumanPlayer(gameUI, OPPONENT));
+        board = new Board(3);
```

Comment:
Instead of using the number `3` here it would be nice if you referred to it with a constant in your test class, i.e.:

`private String 3_x_3 = 3`

that way your `Board` initialisation will look so much nicer:

`board = new Board(3_X_3);`